infra/resources/modules/commons/api/ts_management/v1/_openapi.yaml

```diff
@@ -273,6 +304,35 @@ components:
       schema:
         type: string
         minLength: 1
+    queryPageSize:
+      name: pageSize
+      in: query
+      required: false
+      schema:
+        type: integer
+        minimum: 1
+        maximum: 100
+        default: 25
+    queryMaximumId:
```

Comment:
**suggestion**: add a description

I'd a description about what means this parameter and how a user can used.

Reply:
Added descriptions in 71323943e0b5b6de4d732d5465c4bafa2973e4b1